tests/test_flask_admin.py

```diff
@@ -82,7 +84,7 @@ def test_add_distro(self):
                 sess['nickname'] = 'pingou'
                 sess['email'] = 'pingou@pingoured.fr'
 
-            output = c.get('/distro/add', follow_redirects=True)
```

Comment:
When we make a GET request against this endpoint, there should not be a redirection, so removing the ``follow_redirects`` ensures that we are not redirected (otherwise it means there is something wrong in the code elsewhere).